src/raftstore/store/config.rs

```diff
@@ -79,6 +82,11 @@ pub struct Config {
     /// When a peer hasn't been active for max_peer_down_duration,
     /// we will consider this peer to be down and report it to pd.
```

Comment:
the peer is considered to be down and is reported to PD. (Please capitalize PD whenever possible.)